src/actions/types.ts

```diff
@@ -4,6 +4,7 @@ export const SET_TOKEN = 'SET_TOKEN'
 
 export interface Token {
     token: JWTToken
+    isPending: boolean
```

Comment:
what do you mean with pending ? Could u pls provide some details about this logic ?

Reply:
The token is not filled yet. We're listening on `onAuthStateChanged` function to fill it. `isPending` lets me know when the token is filled. It usefull to avoid flash effects

Reply:
wouldn't that be the same to verify `if(!token)` to achieve this ?

Reply:
Nop

You're token is not initialized yet. Let me show with a screencast

Reply:
I think I understood, I'd be pleased to see a screencast if you have the time, otherwise I'll test it locally